### PR TITLE
Improve the About page

### PR DESCRIPTION
**Pull request by Natim**
Source: https://twitter.com/nhoizey/status/667585190651297793

We should emphasis more that we're a Mozilla team.

### CODE REVIEW

**Comment by Natim** [comment]
Is that what you meant? r? @n1k0 @nhoizey

File: content/pages/about-en.rst [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+`Mozilla <https://www.mozilla.com/>`_:
+`n1k0 <https://nicolas.perriault.net/>`_,
+`Mathieu <http://mathieu-leplatre.info>`_, `Tarek <http://ziade.org/>`_,
+`Rémy <https://twitter.com/Natim>`_ and `Alexis <http://notmyidea.org>`_.
+
+After some time spend accumulating experience without writting about
+it, we decided to start this blog, where we write in French and in
+English, about our explorations and reflections.
+
+We encourage you, the reader, to interact criticise and give feedback
```
Comment (almet) [inline content/pages/about-en.rst]: It's obvious that "you" is the reader here. I would drop it.

File: content/pages/about-en.rst [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+about what we write in order for us to improve the outcome of what we
+are trying to build with you.
```
Comment (almet) [inline content/pages/about-en.rst]: This sentence seems a bit heavy. What about: "Don't hesitate to interact and criticize: we want to build this with the help of the broader community (you!)".

**Comment by almet** [comment]
r+wc

**Comment by nhoizey** [comment]
Perfect!

**Comment by Natim** [comment]
It has been published online.

File: content/pages/about-en.rst [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+:lang: en
+:url: about
+
+We are the "Cloud-services" french team at
+`Mozilla <https://www.mozilla.com/>`_:
+`n1k0 <https://nicolas.perriault.net/>`_,
+`Mathieu <http://mathieu-leplatre.info>`_, `Tarek <http://ziade.org/>`_,
+`Rémy <https://twitter.com/Natim>`_ and `Alexis <http://notmyidea.org>`_.
+
+After some time spend accumulating experience without writting about
```
Comment (leplatrem) [inline content/pages/about-en.rst:14]: typo: writing

**Pull request merged**